Labber_util.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Jun 20 15:00:15 2022
+
+@author: owner
+"""
+
+import Labber as lb
+import os
+def log_exists(log_name):
+    #create a temporary log:
+    lLog = [dict(name='name', vector = False)]
+    templog = lb.createLogFile_ForData('temp', lLog)
+    path = templog.getFilePath(None)
+    folder = os.path.dirname(path)
+    log_name = f'{log_name}.hdf5'
+    new_file_path = os.path.join(folder, log_name)
+    return(os.path.exists(new_file_path))
+
+
+def get_log_name(log_name):
+    i=1
+    log_name_temp = log_name
+    while log_exists(log_name_temp):
+        log_name_temp = f'{log_name}__{i}'
```

Comment:
`log_name_temp` is redundant.
```suggestion
    while log_exists(f'{log_name}__{i}'):
```
and return `f'{log_name}__{i}` at the end.
Also, `'{log_name}__{i}'` should probably be a constant
```python
# declaration
LOG_NAME_FORMAT = '{log_name}__{counter}`
# ...

# usage
LOG_NAME_FORMAT.format(log_name="blabla", counter=3)
```

Reply:
this does not produce the same behavior as the original code:
I want that to get just log_name (without any integer suffix) in the case that does not already exist in the database
I did fix  constant format thing.